demos/_includes/DemoLookup.php

```diff
@@ -56,7 +57,7 @@ protected function initQuickNewRecord(): void
                 $chain->dropdown('set value', $row['value'])->dropdown('set text', $row['title']);
                 $ret[] = $chain;
 
-                return $ret;
+                return new JsBlock($ret);
```

Comment:
TODO create JsBlock first and make JsBlock::addStatement() public and use it as much as possible